install-agora-git.sh

```diff
@@ -187,4 +190,6 @@ EOF
 [ -x /usr/local/etc/profile ] || (sudo ln -sf /etc/profile /usr/local/etc/profile)
 [ -x /usr/local/etc/profile.d/GNUstep.sh ] || (sudo ln -sf `gnustep-config --variable=GNUSTEP_MAKEFILES`/GNUstep.sh /usr/local/etc/profile.d/GNUstep.sh)
 
+make_services
```

Comment:
Why run `make_services` twice in this script?

Reply:
One isn't being run, it's being added to the .xinitrc to run before starting GNUStep.

Reply:
Ah, of course! I see it now. Nevermind. Carry on.